components/x-audio/src/redux/__tests__/player-logic.test.js

```diff
@@ -33,6 +33,24 @@ describe('actions and reducer', () => {
 		expect(updatedState).toMatchSnapshot();
 	});
 
+	test('Ended action sets ended to false', () => {
+		const updatedState = runActions(initialState, actions.play(), actions.ended());
+
+		expect(updatedState).toMatchSnapshot();
+	});
+
+	test('Playing audio after previous has ended resets ended state', () => {
```

Comment:
@benbarnett I'm not sure which part this should be done. I'm sorry if this is the wrong place to mention.
We need to reset `currentTime` as well?

Reply:
I think we're ok for the `currentTime` as that bit is controlled by the audio element itself.... i.e as soon as the audio starts playing again, it will fire a new event, which will take control of the `currentTime`.

I couldn't see any side effects in testing, but this is worth double checking. Thanks.